docs/_docs/usage.md

```diff
@@ -7,8 +7,8 @@ The Jekyll gem makes a `jekyll` executable available to you in your terminal.
 
 You can use this command in a number of ways:
 
-* `jekyll new` - Creates a new Jekyll site with default gem-based theme
-* `jekyll new --blank` - Creates a new blank Jekyll site scaffold
+* `jekyll new .` - Creates a new Jekyll site in present working directory with default gem-based theme. The `.` can be replaced with a path. The path will be created if it does not exist.
+* `jekyll new --blank .` - Creates a new blank Jekyll site scaffold in the present working directory.
```

Comment:
Similarly, here as well..
```suggestion
* `jekyll new some/path --blank` - Creates a new blank Jekyll site scaffold at path `./some/path`.
```

Reply:
Let's make it consistent with the previous line (either `jekyll new [path] --blank` or `jekyll new PATH --blank`)

Reply:
I had suggested `jekyll new some/path` in the previous line, so `jekyll new some/path --blank` here..
@MSVO If you choose to go with the placeholder suggestion, opt for `jekyll new PATH` and
`jekyll new PATH --blank` for consistency with output from command `jekyll help new`